slack-archive/src/main/java/juja/microservices/slack/archive/repository/ArchiveRepository.java

```diff
@@ -1,14 +1,17 @@
 package juja.microservices.slack.archive.repository;
 
-import juja.microservices.slack.archive.model.Channel;
-import juja.microservices.slack.archive.model.Message;
+import juja.microservices.slack.archive.model.entity.Channel;
+import juja.microservices.slack.archive.model.entity.Message;
 
 import java.util.List;
 
 public interface ArchiveRepository {
```

Comment:
Maybe better  extract logic for messages to another repository
and rename this repo to slackchannelrepository ?

what do you think?

Reply:
Oh you suggest it too)

Reply:
done!